.lagoon.yml

```diff
@@ -11,16 +11,28 @@ tasks:
         service: app
 
 environments:
-  master:
+  production:
     routes:
       - storybook:
         - "www.ripple.sdp.vic.gov.au":
             tls-acme: 'false'
             insecure: Allow
             hsts: max-age=31536000
-        - "ripple.sdp.vic.gov.au":
+    monitoring_urls:
+      - "www.ripple.sdp.vic.gov.au"
```

Comment:
Do we need to have monitoring set up for storybook?

Reply:
Well storybook is the service that the routes point to so yes, but I'm guessing you might mean do we need to monitor the app route then? For that I don't know.

Reply:
Sorry - I meant as in since Storybook isn't really a consumer facing site.

Reply:
I get you now. @kurtfoster Any thoughts?

Reply:
@Yuuupie / @FleetAdmiralButter , I don't think we need to monitor storybook.